bftengine/src/bftengine/MsgsCommunicator.cpp

```diff
@@ -68,8 +68,11 @@ uint32_t MsgsCommunicator::numOfConnectedReplicas(uint32_t clusterSize) {
 }
 
 bool MsgsCommunicator::isUdp() {
+#ifdef USE_COMM_UDP
```

Comment:
Could the dynamic check fail without this ifdef?

Reply:
yes, because `PlainUDPCommunication` may be ifdef'ed out and not defined at all